package.json

```diff
@@ -141,6 +141,9 @@
   "xo": {
     "ignore": [
       "cli.js"
-    ]
+    ],
+		"rules": {
+			"no-loop-func": 1
```

Comment:
I always try to avoid functions in functions. I will need to add another `_` method.